README.md

```diff
@@ -258,43 +258,28 @@ See an extensive and up-to-date description of the `bingo` usage below:
 ## Command Help
 
 ```bash mdox-exec="bingo --help" mdox-expect-exit-code=2
-bingo: 'go get' like, simple CLI that allows automated versioning of Go package level binaries (e.g required as dev tools by your project!)
-built on top of Go Modules, allowing reproducible dev environments. 'bingo' allows to easily maintain a separate, nested Go Module for each binary.
-
+Bingo is a command line tool for Like `go get`  but for Go tools! 
+CI Automating versioning of Go binaries in a nested, isolated Go modules.
```

Comment:
I wished we could work on description in separate PR e.g I don't get "CI Automatic versioning.." sentence.